compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -81,6 +82,7 @@ void buildIREEPrecompileTransformPassPipeline(
     }
     }
 
+    IREE::Flow::buildMeshToFlowConversionPassPipeline(passManager);
```

Comment:
this should be in the common input conversion pass pipeline

Reply:
I added the `IREE::` prefix in all places.